templates/named.conf.local.forward.j2

```diff
@@ -0,0 +1,17 @@
+## {{ ansible_managed }}
+{% for fwd_zone in bind_config_forward_zones %}
+######## {{ fwd_zone.name }} ({{ fwd_zone.zones|count }} zones)
+{% for zone in fwd_zone.zones %}
+zone "{{ zone }}" {
+    type forward;
```

Comment:
this would also cache the results. What do you think about adding additionally a `forward only;` to the config?